OpenSim/Simulation/Model/Geometry.cpp

```diff
@@ -218,7 +218,7 @@ void Mesh::extendFinalizeFromProperties() {
     if (!isObjectUpToDateWithProperties()) {
         const Component* rootModel = nullptr;
         if (!hasOwner()) {
-            std::cout << "Mesh " << get_mesh_file() << " not connected to model..ignoring\n";
+            std::cout << "Mesh " << get_mesh_file() << " not connected to model..ignoring" << std::endl;
```

Comment:
This is to flush the stream so that users will see the messages. Otherwise, the messages might not be printed (until the stream is eventually flushed somewhere else).